content/en/docs/concepts/configuration/configmap.md

```diff
@@ -212,7 +212,8 @@ As a result, the total delay from the moment when the ConfigMap is updated to th
 when new keys are projected to the Pod can be as long as the kubelet sync period + cache
 propagation delay, where the cache propagation delay depends on the chosen cache type
 (it equals to watch propagation delay, ttl of cache, or zero correspondingly).
-
+{{< note >}}
+ConfigMaps consumed as environment variables are not updated automatically and require a pod restart. 
```

Comment:
Incomplete short code, should be closed by a `{{< /note >}}`. However, I'd suggest we remove the shortcode completely.